apps/nextjs-website/src/pages/_document.tsx

```diff
@@ -1,10 +1,32 @@
 import { Html, Head, Main, NextScript } from 'next/document';
 
 export default function Document() {
+  const matomo = `
```

Comment:
We'd like to make sure matomo is running only on certain condition (e.g. when a variable has a defined value).
Could you update the PR handling that?

The ratio is: if we are on production environment, then we want to track with matomo; otherwise we don't need to track anything with matomo